src/components/Reusable/index.js

```diff
@@ -0,0 +1,23 @@
+import Button from "./Button";
+import Card from "./Card";
+import Form from "./Form";
+import Header from "./Header";
+import Icon from "./Icon";
+import Input from "./Input";
+import Label from "./Label";
+import Main from "./Main";
+import NavBar from "./NavBar";
+import Text from "./Text";
+
+export {
```

Comment:
you can directly export components, no need to import + export here

Reply:
made the changes. thanks for pointing out.